java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -528,6 +703,7 @@ public boolean isClean() {
     public long getHostMemorySize() {
       long total = 0;
       if (valid != null) {
+        //TODO: fix for lists
```

Comment:
This needs to be done, but can be done in a follow on if necessary.  We should also make sure that it happens for the device side, which I think is only looking at the top level buffers too.

Reply:
Did so something here that should account for children in lists. please take a look if it seems ok.